### PR TITLE
Document the cnf tests image.

### PR DESCRIPTION
**Pull request by fedepaol**

### CODE REVIEW

**Comment by fedepaol** [comment]
/hold
Giving @SchSeba time for a review

File: cnf-tests/README.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig -e IMAGE_REGISTRY my.local.registry:5000/ quay.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
+```
+
+## Test Reports
+
+The tests have two kind of outputs
+
+### JUnit test output
+
+A junit compliant xml is produced by passing the `-juni`t parameter together with the path where the report is dumped:
```
Comment (SchSeba) [inline cnf-tests/README.md:122]: `-juni`t -> `--junit`

File: cnf-tests/README.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+```bash
+    docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig quay.io/openshift-kni/cnf-tests /usr/bin/test-run.sh -junit /path/to/junit
```
Comment (SchSeba) [inline cnf-tests/README.md:125]: I think you need to add here another volume to take the junit file from inside the container

File: cnf-tests/README.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+```
+
+### Test Failure Report
+
+A report with informations about the cluster state (and resources) for troubleshooting can be produced by passing the `-report` parameter together with the path where the report is dumped:
+
+```bash
+    docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig quay.io/openshift-kni/cnf-tests /usr/bin/test-run.sh -report /path/to/report
```
Comment (SchSeba) [inline cnf-tests/README.md:133]: ditto

File: cnf-tests/README.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+A junit compliant xml is produced by passing the `-juni`t parameter together with the path where the report is dumped:
+
+```bash
+    docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig quay.io/openshift-kni/cnf-tests /usr/bin/test-run.sh -junit /path/to/junit
+```
+
+### Test Failure Report
+
+A report with informations about the cluster state (and resources) for troubleshooting can be produced by passing the `-report` parameter together with the path where the report is dumped:
```
Comment (SchSeba) [inline cnf-tests/README.md:130]: `-report` -> `--report`

File: cnf-tests/README.md (hunk 3 above)
Comment (SchSeba) [reply cnf-tests/README.md:133]: -report -> --report

File: cnf-tests/README.md (hunk 1 above)
Comment (slintes) [inline cnf-tests/README.md:122]: `-juni`t is a bit messed up :)

Comment (SchSeba) [reply cnf-tests/README.md:122]: Yep :)

**Pull request merged**